server/graphql/general/src/queries/tests/names.rs

```diff
@@ -175,6 +177,10 @@ mod graphql {
             );
             assert_eq!(is_visible, Some(false));
             assert_eq!(is_system_name, Some(true));
+            // assert_eq!(
+            //     r#type,
+            //     Some(EqualFilter::equal_to(NameType::Store))
+            // );
```

Comment:
Why is it commented out?

Reply:
ah. yes - had trouble getting that working.
I think that I've figured it out now: 0d1a3bf53